modules/Bio/EnsEMBL/Test/TestUtils.pm

```diff
@@ -485,7 +485,10 @@ sub has_apache2_licence {
     return __PACKAGE__->builder->ok(1, "$file has a no critic (RequireApache2Licence) directive");
   }
   if($found_copyright && $found_url && $found_warranties && $found_sanger_embl_ebi_year && $found_embl_ebi_year) {
```

Comment:
I would prefer a single if statement checking found_(copyright,url,warranties) and, nested into it, two ifs checking sanger_embl_ebi_year^embl_ebi_year and no_affiliation

Reply:
Agree with @avullo . As the if($found_copyright && $found_url && $found_warranties) is common in both the if blocks, it could be nested.